lib/SimpleSAML/Metadata/MetaDataStorageHandlerPdo.php

```diff
@@ -142,6 +142,48 @@ public function getMetadataSet($set)
         return $metadataSet;
     }
 
+    /**      
+     * Retrieve a metadata entry.
+     *      
+     * @param string $entityId The entityId we are looking up.
+     * @param string $set The set we are looking for metadata in.
+     *
+     * @return array An associative array with metadata for the given entity, or NULL if we are unable to
+     *         locate the entity.
+     */
+    public function getMetaData($entityId, $set)      
+    {       
+        assert('is_string($entityId)');
+        assert('is_string($set)');
```

Comment:
assert should not have string arguments (since PHP 7.2). Should be like:
```
        assert(is_string($entityId));
        assert(is_string($set)); 
```